solr/contrib/ltr/src/test/org/apache/solr/ltr/feature/TestFeatureLtrScoringModel.java

```diff
@@ -46,8 +46,8 @@ public void getInstanceTest() throws FeatureException
   {
 
     final Map<String,Object> map = new HashMap<String,Object>();
-    map.put(ManagedFeatureStore.NAME_KEY, "test");
-    map.put(ManagedFeatureStore.CLASS_KEY, OriginalScoreFeature.class.getCanonicalName());
+    map.put("name", "test");
```

Comment:
Oops, meant to but forgot to comment. I agree in general it would be good to not have these magic strings and tests can usually easily access package-visible constants of the classes that they test. This test however is not in the same package as the ManagedFeatureStore and so it cannot see any package-visible ..._KEY constants. And to give the constant public visibility (including inclusion in javadocs) just so that these tests can see it seems kinda wrong. The 'for later' solution I presume is for the tests to be restructured and then the need for this class here in this package to 'see' the ..._KEY would go away. That kind of make sense?